src/fermilib/utils/_plane_wave_hamiltonian.py

```diff
@@ -26,6 +26,33 @@
 from projectq.ops import QubitOperator
 
 
+def wigner_seitz_length_scale(wigner_seitz_radius, n_particles, dimension):
+    """Function to give length_scale associated with Wigner-Seitz radius.
```

Comment:
That is not a convention we follow anywhere else in FermiLib as far as I can tell. To the contrary, the majority of our code has the summary doc string immediately after the function line.